src/main/python/ayab/ayab.py

```diff
@@ -18,8 +18,8 @@
 #    https://github.com/AllYarnsAreBeautiful/ayab-desktop
 """Provides a graphical interface for users to operate AYAB."""
 
+from __future__ import annotations
 from fbs_runtime.application_context.PySide6 import ApplicationContext
```

Comment:
I _thiiiink_ this import is redundant with the import in main, since GuiMain inherits from Main. But kinda-sorta out of scope for this PR IMO so I've left it